### DISCUSSION TITLE
Possibly utilize a numbered list instead of "bullets"

### DISCUSSION THREAD

**Discussion by looneyapache** [TfTHacker/DashboardPlusPlus#5] [General]
If there was a way to utilize a numbered list instead of "bullets" on the dashboard, that would be interesting.

Thank you!

**Comment by TfTHacker** [comment]
It depends where you want to number. but for the most part it should be possible. may require some CSS tweeks. Though this worked without any changes:

![CleanShot 2022-11-01 at 05 41 43@2x](https://user-images.githubusercontent.com/69121180/199160360-5268ddc1-98df-4be8-bc7c-caa340388c78.png)

**Comment by looneyapache** [comment]
My mistake; I never intended for numbers to completely replace bullets.

Instead, I considered using numbered bullets similar to an index or table
of contents for my use case.


Using sub bullets as numbered lists is good idea too :).

Thank you!!!
